### PR TITLE
first assignment

### PR DESCRIPTION
**Pull request by emtec-jitendra-patil**
This is my first assignment of SQL

### CODE REVIEW

File: SQL Assignment 1.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+--);
+
+--CREATE TABLE student(
+--	id INTEGER PRIMARY KEY,
+--	name VARCHAR(20) NOT NULL,
+--	address VARCHAR(30) NOT NULL,
+--	phone bigint UNIQUE NOT NULL,
+--	email VARCHAR(20) UNIQUE NOT NULL,
+--	age INTEGER NOT NULL,
+--	FOREIGN KEY(id) references school(id)
```
Comment (rishikesh-dhanawade-emtec) [inline SQL Assignment 1.txt:28]: Column name missing

File: SQL Assignment 1.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+		---Assigment 1---
+--create 'student_db' database
+--create following tables in database with proper data types
+--student: id (PK), name, address, phone, email, age, schoolId (FK)
+--school: id (PK), name, address, principal, phone
+--create relationship as shown in the table schema
+--add some dummy data
```
Comment (rishikesh-dhanawade-emtec) [inline SQL Assignment 1.txt:7]: add dummy data - this requirement is missing in code

**Comment by emtec-jitendra-patil** [comment]
Sorry sir i forgot to add dummy data

**Review by rishikesh-dhanawade-emtec — CHANGES_REQUESTED** [review]
resolve the comments and submit it again

File: SQL Assignment 1.txt (hunk 1 above)
Comment (emtec-jitendra-patil) [reply SQL Assignment 1.txt:28]: Sorry sir , I forgot to add column name

> Column name missing

**Comment by emtec-jitendra-patil** [comment]
> resolve the comments and submit it again

ok sir

**Review by rishikesh-dhanawade-emtec — APPROVED** [review]
approved

**Pull request merged**